students/RandyKline/lesson03/l3_slicer.py

```diff
@@ -0,0 +1,57 @@
+# Lesson 3 - Slicing
+# Randy Kline
+
+a_string = "abcdefghijklmnopqrstuvwxyz"
+a_tuple = (13, 12, 5, 32, 2, 54);
+
+def exchange_first_last(seq):
+  seq_first = seq[0]
+  seq_middle = seq[1:-1]
+  seq_last = seq[-1]
+  if type(seq) is str:
+    a_new_sequence = (seq_last+seq_middle+seq_first)
+    return a_new_sequence
+  else:
+    a_new_sequence = (seq_last,seq_middle,seq_first)
+  return a_new_sequence
```

Comment:
the rest of your functions use slicing; can you change this function to also _only_ use slicing? (hint: this would also be a one liner and not type specific).